src/main/java/touro/chess/PawnPiece.java

```diff
@@ -22,54 +22,61 @@ public PawnPiece(Location location, PieceColor color) {
 
```

Comment:
I prefer the readability on the last code I commit. 
I felt like it was easier to read through and understand.